doc/destructors.rst

```diff
@@ -88,7 +88,7 @@ Lifetime-tracking hooks
 =======================
 
 The memory management for Nim's standard ``string`` and ``seq`` types as
-well as other standard collections is performed via so called
+well as other standard collections are performed via so-called
```

Comment:
Is 'are' actually correct here? I'm not good at English grammar, but the sentence reads as "The memory management for ... and ... as well as ... **is** performed" since "is" refers to "memory management" here.

Reply:
Correct. fixed 👍